assets/src/functions/hydrateBlock.js

```diff
@@ -13,7 +13,9 @@ export const hydrateBlock = (blockName, Component, csrAttributes = {}) => { // e
   blocks.forEach(
     blockNode => {
       const attributes = JSON.parse(blockNode.dataset.attributes);
-      hydrate(<Component {...attributes} {...csrAttributes} />, blockNode);
+      if(blockNode.length) {
```

Comment:
We should avoid run the `hydrate` function whenever the target container is not a DOM element, in other words an empty NodeList<>.